docs/tech-spec.md

```diff
@@ -0,0 +1,493 @@
+* ASBIEs are grouped by `Property Term Qualifier(s)` + `Property Term` + `Associated Object Class Term Qualifier(s)` + `Associated Object Class` as RDFS Properties
+
+
+#### De-duplication
+
+The above grouping rules may lead to deduplication of several CEFACT BIEs into single class or property. For example both _SupplyChain_Consignment_ and _Referenced_SupplyChain_Consignment_ BIEs get merged into one _Consignment_ class.
+
+Such deduplication is necessary to make the RDFS modelling guidelines to be unambiguous.
+
+#### Primary identifier mapping
+
+Some CEFACT BIEs have explicit primary identifier property, for example Referenced_SupplyChain_Consignment.Identification.Identifier. These properties are omitted in the RDF vocabulary, as RDF data model makes the primary identifier an inherent attribute for each entitiy in the graph.
+
+### UN/CEFACT metadata
+
+We provide and publish the machine-readable RDF representation of The CEFACT Buy-Ship-Pay RDM Business Information Elements, preserving their types, inheritance heirarchy and metadata. All rdfs classes and properties in edi3 vocabulary are linked with corresponding BIEs by its identifier. This link can be used to implement a software which automatically maps CEFACT RDM messages to RDF format. So that interoperability between existing systems which use CEFACT RDM and new Linked Data based systems is preserved.
+
+The example rdfs property from the edi3 vocabulary, with linked CEFACT RDM BIEs:
+```json
+{
+  "@id": "edi3:consignorTradeParty",
+  "@type": "rdfs:Property",
+  "rdfs:domain": "edi3:Consignment",
+  "rdfs:range": "edi3:Party",
+  "edi3:cefactElementMetadata": [
+    {
+      "@id": "cefact:Referenced_SupplyChain_Consignment.Consignor.Trade_Party",
+      "@type": "edi3:AssociationBIE", 
+      "edi3:cefactUNId": "cefact:UN01011054",
+      "edi3:cefactBieDomainClass": "cefact:Referenced_SupplyChain_Consignment.Details",
+      "edi3:cefactBusinessProcess": "Buy-Ship-Pay"
+    },
+    {
+      "@id": "cefact:SupplyChain_Consignment.Consignor.Trade_Party",
+      "@type": "edi3:AssociationBIE", 
+      "edi3:cefactUNId": "cefact:UN01004212",
+      "edi3:cefactBieDomainClass": "cefact:SupplyChain_Consignment.Details",
+      "edi3:cefactBusinessProcess": "Buy-Ship-Pay"
+    },
+  ]
+}
+```
+ 
+### Business domain granularity
+
+The vocabulary terms are annotated with the logical business domain which this term belongs to:
+
+```json
+{
+  "@id": "edi3:consignorTradeParty",
+  "@type": "rdfs:Property",
+  "edi3:businessDomain": "Trade"
+}
+```
+
+TODO: The formal process of assigning the business domain to the vocabulary terms is to be decided.
+
+### Versioning
+
+The vocabulary is updated every 6 months, following the maintainance cycle of the CEFACT BSP RDM.
+Each BIE is annotated with the date when it was created, current active\deprecated status, and the date of deprecation.
+
+```json
```

Comment:
@kshychko , @fak3 I don't think we're doing this any longer, are we?

Reply:
Are you proposing not to do versioning, or annotation? Or you just state that current vocabulary inherited from edi3 work does not have version annotations?

Reply:
The latter - https://service.unece.org/trade/uncefact/vocabulary/uncefact/ doesn't seem to have it

Reply:
Yes, this have not been implemented